qiskit/circuit/classical/expr/expr.py

```diff
@@ -345,6 +345,14 @@ class Op(enum.Enum):
         """Zero-padding bitshift to the left.  ``lhs << rhs``."""
         SHIFT_RIGHT = 13
         """Zero-padding bitshift to the right.  ``lhs >> rhs``."""
+        ADD = 14
+        """Addition. ``lhs + rhs``."""
+        SUB = 15
+        """Subtraction. ``lhs - rhs``."""
+        MUL = 16
+        """Multiplication. ``lhs * rhs``."""
+        DIV = 17
+        """Division. ``lhs / rhs``."""
```

Comment:
Can you add a paragraph of prose at the end of the class-level docstring mentioning these and the allowed types?

Reply:
Done in f780d9a